src/str_macros.jl

```diff
@@ -63,12 +63,12 @@ end
 
 immutable OpDefExpr
     op_sym::Symbol
-    label_args::Union(Symbol, Expr)
+    label_args::Symbol
     lhs_type::Symbol
-    rhs::Union(Symbol, Expr)
+    rhs::Symbol
 end
 
-OpDefExpr(ods::OpDefStr) = OpDefExpr(symbol(ods.op_name), parse(ods.label_args), symbol(ods.lhs_type), parse(ods.rhs))
+OpDefExpr(ods::OpDefStr) = OpDefExpr(Symbol(ods.op_name), Symbol(parse(ods.label_args)), Symbol(ods.lhs_type), Symbol(parse(ods.rhs)))
```

Comment:
Hmm...why force these to be `Symbol`s, when before they could be `Symbol`s or `Expr`s?